backend/package.json

```diff
@@ -32,6 +32,7 @@
     "nodemon": "^3.1.0",
     "supertest": "^6.3.3",
     "ts-jest": "^29.1.2",
+    "ts-node": "^10.9.2",
```

Comment:
issue(blocking): no need to add it here since it already exist in the dependencies section.